app/cafe.py

```
@@ -0,0 +1,29 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        name = visitor.get("name")
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{name} is not vaccinated")
+
+        vaccine = visitor.get("vaccine")
+        expiration_date = vaccine.get("expiration_date", date(2000, 1, 1))
```

Issue: The use of `vaccine.get('expiration_date', date(2000, 1, 1))` sets a default expiration date far in the past if the key is missing. This may not be the intended behavior according to the task requirements or checklist. If the checklist requires raising an error when the expiration date is missing, you should explicitly check for its presence and handle it accordingly, rather than silently defaulting to an expired date.